src/components/facet-line/FacetLineFilters.tsx

```diff
@@ -60,7 +60,7 @@ const FacetLineFilters: React.FunctionComponent<FacetLineFiltersProps> = ({
                   value: ""
                 }}
                 options={formatValuesToDropdown(name, values)}
-                ariaLabel={name}
+                ariaLabel={t(`facet${upperFirst(name)}Text`)}
```

Comment:
This creates a tricky situation for managing translations as the code does not expose any information about what strings we can expect. This also shows by the fact that no facet translation controls are added in Storybook.

As I see it the possible facet names are limited by [the `FacetField` enumeration](https://github.com/reload/dpl-react/blob/main/src/core/dbc-gateway/generated/graphql.tsx#L306-L306). 


If you want to take a stab at it consider creating a function for generating translations for facet names and use TypeScript to ensure that all values are covered and translations are explicit but I do not think it is necessary to move this PR along.